src/functions-reference/higher-order_functions.Rmd

```diff
@@ -414,6 +414,8 @@ solver, and length of each vector in the output,
 
 *   number of solution times and number of vectors in the output,
 
+## 
```

Comment:
Will this be filled in or is it an error?

Reply:
definitely an error, likely I didn't save properly. Just pushed a fix.